### PR TITLE
Add authorization

### LINKED ISSUE #80: Add authorization to backend endpoints

### PR DESCRIPTION
**Pull request by VegardOztan**
Closes #80

### CODE REVIEW

**Review by GodVenn — COMMENTED** [review]
Maybe adding some comments to describe certain sections would be helpful in the future, for example the fusion token cache?

File: backend/api/Startup.cs [hunk 1]
```diff
@@ -26,23 +29,56 @@ public Startup(IConfiguration configuration)
...
-                        .AllowAnyMethod()
-                        .SetIsOriginAllowedToAllowWildcardSubdomains();
+                    builder.WithOrigins("http://localhost:3000", "https://*.equinor.com")
+                    .AllowAnyHeader()
+                    .AllowAnyMethod()
+                    .SetIsOriginAllowedToAllowWildcardSubdomains();
                 });
             });
             services.AddControllers();
             services.AddSwaggerGen(c =>
```
Comment (GodVenn) [inline backend/api/Startup.cs:56]: Could comment "Add authentication to swagger"

File: frontend/cypress/support/auth.ts [hunk 2]
```diff
@@ -5,16 +5,14 @@ const SERVER_URL = Cypress.env('AUTH_URL') || 'http://localhost:8080'
...
- * or if we are able to store tokens of all users we need (or all users at all)
- */
+/* Gets the access token from local cache which was stored on login. */
 export const getToken = (): string => {
     const fusionStorageJson = localStorage.getItem(CACHE_ENTRY)
     if (fusionStorageJson === null) {
         throw new Error('Could not find auth token in local storage')
     }
     const fusionStorage = JSON.parse(fusionStorageJson)
-    const token = fusionStorage[`${CACHE_ENTRY}:8829d4ca-93e8-499a-8ce1-bc0ef4840176:TOKEN`]
```
Comment (GodVenn) [inline frontend/cypress/support/auth.ts:17]: What is the GUID here?

File: frontend/src/api/apiClient.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import { config } from '../config'
+
+export class apiBackend {
+    /* Gets the access token from local cache which was stored on login. */
```
Comment (GodVenn) [inline frontend/src/api/apiClient.ts:4]: Maybe mention that login happens in fusion and not in our app

File: frontend/cypress/support/auth.ts (hunk 2 above)
Comment (VegardOztan) [reply frontend/cypress/support/auth.ts:17]: It's the client ID. I'll change this to match what we are doing in `frontend/src`

**Review by GodVenn — APPROVED** [review]

**Pull request merged**